mlflow/store/artifact/models_artifact_repo.py

```diff
@@ -155,8 +155,11 @@ def log_artifact(self, local_file, artifact_path=None):
             artifact_path: Directory within the run's artifact directory in which to log the
                 artifact.
         """
+        if self.is_logged_model_uri:
+            return self.repo.log_artifact(local_file, artifact_path)
         raise ValueError(
-            "log_artifact is not supported for models:/ URIs. Use register_model instead."
+            "log_artifact is not supported for models:/<model_name>/<model_version> URIs. "
+            "Use register_model instead."
```

Comment:
```suggestion
            "log_artifact is not supported for models:/<name>/<version> URIs. "
            "Use register_model instead."
```

nit:  I think `model_` is redundant. Let's remove it.